downstream/modules/platform/con-gw-understanding-authenticator-mapping.adoc

```diff
@@ -0,0 +1,95 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="gw-understanding-authenticator-mapping"]
+
+= Understanding authenticator mapping
+
+Authentication:: validates a user’s identity, typically through a username and password or a trust system.
+
+Authorization:: determines what an authenticated user can do once they are authenticated. 
+
+In {PlatformNameShort}, authenticators manage authentication, validating users and returning details such as their username, first name, email, and group memberships (for example, LDAP groups). Authorization comes from the authenticator’s associated maps.
+
+During the authentication process, once a user has authenticated, the authorization system starts with a default set of permissions in memory. Then sequentially, the authenticator maps are processed and adjust permissions based on their trigger conditions. Once all the authenticator's maps are processed, the in-memory representation of the users permissions are reconciled with the users existing permissions. 
```

Comment:
Same as above (use "once" to mean "one time", per IBM; not a replacement for "after" or "when"). Will leave this up to you.

```suggestion
During the authentication process, after a user has authenticated, the authorization system starts with a default set of permissions in memory. Then sequentially, the authenticator maps are processed and adjust permissions based on their trigger conditions. When all the authenticator's maps are processed, the in-memory representation of the user's permissions are reconciled with the user's existing permissions. 
```
Assuming users in this case is possessive and singular. If I've assumed incorrectly, please disregard.

Reply:
Fixed.